run.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 }
 
 function run_tests() {
-   eval "./mvnw -Dtest=$TESTS  test"
+   eval "./mvnw -Dtest=$TESTS -DPLATFORM_TYPE=$PLATFORM test"
```

Comment:
But isn't this the case for SERVER_URI as well? I mean, if you don't run the setup scripts that doesn't get set.